utils/firestore.js

```diff
@@ -1436,21 +1436,30 @@ const getBiospecimenCollectionIdsFromBox = async (requestedSite, boxId) => {
  * query the biospecimen collection for documents that match the healthcare provider and collectionIds in collectionIdArray
  * @param {number} requestedSite - site code of the site 
  * @param {string} boxId - boxId of the box
+ * Firestore 'in' operator limit with two .where() clauses is 15 (30 with one .where() clause).
```

Comment:
Should we define the return of this function?

Reply:
Good call - done. Thanks